Lib/test/test_socket.py

```diff
@@ -946,6 +946,37 @@ def testWindowsSpecificConstants(self):
         socket.IPPROTO_L2TP
         socket.IPPROTO_SCTP
 
+    @unittest.skipUnless(sys.platform == 'darwin', 'macOS specific test')
+    @unittest.skipUnless(support.IPV6_ENABLED, 'IPv6 required for this test')
```

Comment:
Alas, #19603 has landed recently and moved IPV6_ENABLED to test.support.socket_helper.  Update and rebase against the current master HEAD.

Reply:
Thanks, will do ASAP.

Reply:
Fixed in ebfde1f29f11a2041ba38757ba9e9a87fd261ec4